connect/credentials.go

```diff
@@ -39,6 +40,11 @@ func serviceCredentialsFile(service string) string {
 	return filepath.Join(CFG.FsRoot, defaulCredentialsDir, service)
 }
 
+func (c Credentials) authHeader() string {
+	auth := c.Username + ":" + c.Password
+	return "Basic " + base64.StdEncoding.EncodeToString([]byte(auth))
```

Comment:
why do you generate the header by hand instead of `req.SetBasicAuth()` we had before?

Reply:
> why do you generate the header by hand instead of `req.SetBasicAuth()` we had before?

Good point. I don't remember. I will change it back. Thanks.